calendar_af/wizard/create_local_occasion.py

```diff
@@ -58,10 +58,13 @@ class CreateLocalOccasion(models.TransientModel):
         comodel_name="calendar.appointment.type",
         string="Type",
         required=True,
-        domain="[('channel', '=', 'Local')]"
+        domain="[('channel', '=', 'Local')]",
     )
     channel = fields.Many2one(
-        string="Channel", comodel_name="calendar.channel", related="type_id.channel", readonly=True
+        string="Channel",
```

Comment:
It looks like you use a formatting tool on the code, that's great! 
One thing to consider making a commit that only include formatting and an other commit for the actual change.